README.adoc

```diff
@@ -1,4 +1,4 @@
-= C++ example project scanned on SonarCloud with code coverage using Azure Pipelines
+= C++ example project for generating test coverage with vscoverage on Azure Pipelines
```

Comment:
```suggestion
= C++ example project for generating test coverage with VisualStudio coverage tool on Azure Pipelines
```
IMHO vscoverage isn't a thing, and we can afford to spell the full name of the concept